README.md

```diff
@@ -239,30 +239,31 @@ Benchmark comparison test [details data](https://gist.github.com/membphis/137db9
 
 [![Contributor over time](https://contributor-graph-api.apiseven.com/contributors-svg?repo=apache/apisix)](https://www.apiseven.com/en/contributor-graph?repo=apache/apisix)
 
-## Videos And Articles
-
-- [Apache APISIX: How to implement plugin orchestration in API Gateway](https://www.youtube.com/watch?v=iEegNXOtEhQ)
```

Comment:
We no longer reserve these articles in README?

Reply:
The readme is too long, apisix.apache.org is a better place for them